configuration/components/observatorium.libsonnet

```diff
@@ -79,7 +79,13 @@ local api = (import 'observatorium-api/observatorium-api.libsonnet');
       ],
     },
     traces: {
+      // gRPC OpenTelemetry collector for traces endpoint
       writeEndpoint: obs.tracing.manifests.otelcollector.metadata.name + '-collector:4317',
+
+      // Jaeger V2 HTTP query for traces using Observatorium tenancy pattern
+      templateEndpoint: 'http://jaeger-{tenant}-collector:16686',
+
+      enabled: false,
```

Comment:
`tracing.libsonnet` has enabled flag as well https://github.com/observatorium/observatorium/blob/main/configuration/components/tracing.libsonnet#L10 which controls tracing manifests (jaeger, OTELcol CRs) generation - https://github.com/observatorium/observatorium/blob/main/configuration/components/observatorium.libsonnet#L164 
Could we unify those two enabled fields? 


I don't see where the `enabled` field from this PR is used e.g. in a condition.

Reply:
This enabled field is being used in the draft PR https://github.com/observatorium/api/pull/280/files#diff-d79e05fae1c17a80c15dfd8e05409c355f8eb20c179667763c26ef6351c18fbfR133

See also this comment: https://github.com/observatorium/observatorium/issues/472#issuecomment-1097011831

I'm not a Jsonnet expert but I spent several days trying to access the other `.enabled` or to use an empty `writeEndpoint` field.  Those fields don't seem to be visible from _jsonnet/lib/observatorium-api.libsonnet_.

Reply:
I think we could define the `enabled` field only in `observatorium-api.libsonnet` as the condition where it is used is in that file.

Reply:
@pavolloffay I will defer to you and others for the best pattern to enable/disable in Jsonnet.

I am a Jsonnet beginner.  I saw in _jsonnet/lib/observatorium-api.libsonnet_ the code 

```Jsonnet
                if api.config.traces != {} then
                  [
                    '--traces.write.endpoint=' + api.config.traces.writeEndpoint,
                    '--grpc.listen=0.0.0.0:' + api.config.ports['grpc-public'],
```

... and I assumed that these ports would be closed unless tracing was defined.  At that time I hadn't looked at observatorium/observatorium.

When I looked at the examples in observatorium/observatorium, the _base_, _dev_, and _local_ in _configuration/examples_, I thought that only _local_ had tracing and thus only _local_ should have a _manifests/api-deployment.yaml_ with `--traces.write.endpoint`.  It was confusing to see the API server enabling traces when the collector wasn't deployed.

This PR is my attempt to pass the flag into _observatorium-api.libsonnet_ so that a .jsonnet file can enable or disable tracing.